### PR TITLE
Adds a Pico plush

### PR DESCRIPTION
**Pull request by FernandoJ8**
## About The Pull Request

Adds a plush of Tom Fulp's character Pico, from Pico's School.
https://user-images.githubusercontent.com/80640114/120684363-e2919280-c49e-11eb-92c7-109dc06494ad.mp4
unknown

## Why It's Good For The Game

It was about dang time we honoured our origins and added NG references. Might as well start with the man we got our name from.

## Changelog
:cl:
add: Added the Pico plushie
/:cl:

### CODE REVIEW

**Comment by FernandoJ8** [comment]
I'm a moron.

File: fulp_modules/edits/container_init/machine_init.dm [hunk 1]
```diff
@@ -85,6 +85,10 @@
 	products += list(/obj/item/toy/plush/batong = 3) //Used in toys.dm
 	. = ..()
 
+/obj/machinery/vending/games/Initialize()
+    products += list(/obj/item/toy/plush/pico = 3) //Used in toys.dm as well
+    . = ..()
```
Comment (SgtHunk) [inline fulp_modules/edits/container_init/machine_init.dm:90]: Don't add another Initialize.
Use the one above, its a list.

File: fulp_modules/features/toys/code/toys.dm [hunk 2]
```diff
@@ -24,3 +24,12 @@
 	attack_verb_continuous = list("dust")
 	attack_verb_simple = list("dust")
 	squeak_override = list('sound/effects/supermatter.ogg'= 1)
+
+/obj/item/toy/plush/pico
+	name = "Pico plushie"
```
Comment (SgtHunk) [inline fulp_modules/features/toys/code/toys.dm:29]: lowercase

File: fulp_modules/features/toys/code/toys.dm [hunk 3]
```diff
@@ -24,3 +24,12 @@
...
+	desc = "A plushie of an alarmed middle schooler with orange hair and a green T-Shirt. Maybe somethng's happening at his school."
```
Comment (SgtHunk) [inline fulp_modules/features/toys/code/toys.dm:30]: "somethng"

**Review by SgtHunk — CHANGES_REQUESTED** [review]

File: fulp_modules/features/toys/code/toys.dm (hunk 3 above)
Comment (FernandoJ8) [reply fulp_modules/features/toys/code/toys.dm]: Hm? Is somethng wrong with that line?

Jokes aside all of these should be fixed with the new push

File: fulp_modules/edits/container_init/machine_init.dm [hunk 4]
```diff
@@ -83,7 +83,8 @@
 
 /obj/machinery/vending/games/Initialize()
 	products += list(/obj/item/toy/plush/batong = 3) //Used in toys.dm
-	. = ..()
+    products += list(/obj/item/toy/plush/pico = 3) //Used in toys.dm as well
+    . = ..()
```
Comment (SgtHunk) [inline fulp_modules/edits/container_init/machine_init.dm:87]: its a list
add a comma after the 3 and continue the list

**Review by SgtHunk — CHANGES_REQUESTED** [review]

File: _maps/map_files/SeleneStation/SeleneStation.dmm [hunk 5]
```diff
@@ -37874,9 +37874,10 @@
 	name = "Medical blue corner"
 	},
 /obj/item/book/manual/wiki/medicine,
-/obj/item/toy/plush/lizard_plushie,
+/obj/item/toy/plush/pico,
+/obj/item/toy/plush/pico,
```
Comment (remove32) [inline _maps/map_files/SeleneStation/SeleneStation.dmm:37878]: you still have the lizard replaced here

File: fulp_modules/edits/container_init/machine_init.dm [hunk 6]
```diff
@@ -82,8 +82,8 @@
 	. = ..()
 
 /obj/machinery/vending/games/Initialize()
-	products += list(/obj/item/toy/plush/batong = 3) //Used in toys.dm
-	. = ..()
+	products += list(/obj/item/toy/plush/batong = 3, /obj/item/toy/plush/pico = 3) //Used in toys.dm
+    . = ..()
```
Comment (remove32) [inline fulp_modules/edits/container_init/machine_init.dm:86]: why are you nesting this

**Review by remove32 — CHANGES_REQUESTED** [review]

File: fulp_modules/edits/container_init/machine_init.dm [hunk 7]
```diff
@@ -82,7 +82,7 @@
 	. = ..()
 
 /obj/machinery/vending/games/Initialize()
-	products += list(/obj/item/toy/plush/batong = 3) //Used in toys.dm
+	products += list(/obj/item/toy/plush/batong = 3, /obj/item/toy/plush/pico = 3) //Used in toys.dm
```
Comment (JohnFulpWillard) [inline fulp_modules/edits/container_init/machine_init.dm:85]:
```suggestion
	products += list(/obj/item/toy/plush/batong = 3,
		/obj/item/toy/plush/pico = 3) //Both used in toys.dm
```

Comment (SgtHunk) [reply fulp_modules/edits/container_init/machine_init.dm:85]: trailing comma my beloved

Comment (JohnFulpWillard) [reply fulp_modules/edits/container_init/machine_init.dm:85]: :trollface:

**Comment by FernandoJ8** [comment]
*What's a merge conflict and how do I fix it?
*

**Comment by JohnFulpWillard** [comment]
Where are they on the maps!! Where's Helio and Selene map edits!!!

**Comment by FernandoJ8** [comment]
> 
> 
> Where are they on the maps!! Where's Helio and Selene map edits!!!

To be determined

File: fulp_modules/Z_edits/container_init/machine_init.dm [hunk 8]
```diff
@@ -0,0 +1,141 @@
+/*
```
Comment (JohnFulpWillard) [inline fulp_modules/Z_edits/container_init/machine_init.dm:1]: the shit is this

**Comment by FernandoJ8** [comment]
I swear to god I'm going to fucking die.

**Comment by FernandoJ8** [comment]
I've no clue why the hell machine_init.dm is causing problems, i literally copypasted the lines from the fulstation/fulpstation repository file and changed the lines for my thing

**Comment by EgorDinamit** [comment]
Add interaction with Uzi so you could give him a gun.

File: fulp_modules/Z_edits/container_init/machine_init.dm [hunk 9]
```diff
@@ -82,8 +82,9 @@
 	. = ..()
 
 /obj/machinery/vending/games/Initialize()
-	products += list(/obj/item/toy/plush/batong = 3) //Used in toys.dm
-	. = ..()
+	products += list(/obj/item/toy/plush/batong = 3,
+	/obj/item/toy/plush/pico = 3, //Both used in toys.dm
+	)
```
Comment (EgorDinamit) [inline fulp_modules/Z_edits/container_init/machine_init.dm:87]:
```suggestion
	products += list(/obj/item/toy/plush/batong = 3,
	/obj/item/toy/plush/pico = 3) //Both used in toys.dm
	. = ..()
```

File: fulp_modules/features/toys/code/toys.dm [hunk 10]
```diff
@@ -24,3 +24,12 @@
...
 	squeak_override = list('sound/effects/supermatter.ogg'= 1)
+
+/obj/item/toy/plush/pico
+	name = "pico plushie"
+	desc = "A plushie of an alarmed middle schooler with orange hair and a green T-Shirt. Maybe something's happening at his school."
+	icon = 'fulp_modules/features/toys/icons/toys.dmi'
+	icon_state = "pico"
+	attack_verb_continuous = list("flails at")
+	attack_verb_simple = list("flail at")
+	squeak_override = list('fulp_modules/features/toys/sound/pico.ogg'= 1)
```
Comment (EgorDinamit) [inline fulp_modules/features/toys/code/toys.dm:35]: I'd give him another sound, since the one you used is the classroom's ambience, y'know, apples.

Comment (EgorDinamit) [reply fulp_modules/features/toys/code/toys.dm:35]: Actually never mind, watched the video you attached to the PR's body. Sounds good.

**Comment by FernandoJ8** [comment]
Moved to  https://github.com/fulpstation/fulpstation/pull/212#issue-663856508 to hopefully resolve the borked branch issues

**Comment by FernandoJ8** [comment]
> 
> 
> Add interaction with Uzi so you could give him a gun.

There's a few interactions I want to code, but those will have to come in the future, cause I've no idea how to code them right now

**Pull request closed without merging**